compiler+runtime/CMakeLists.txt

```diff
@@ -96,6 +92,9 @@ set(
 set(
   jank_linker_flags
   #-stdlib=libc++ -lc++abi
+  $<$<PLATFORM_ID:Darwin>:-L/opt/homebrew/lib>
+  $<$<PLATFORM_ID:Darwin>:${CMAKE_BINARY_DIR}/llvm-install/usr/local/lib/libunwind.a>
+  $<$<PLATFORM_ID:Darwin>:${CMAKE_BINARY_DIR}/llvm-install/usr/local/lib/libc++abi.a>
```

Comment:
@jeaye 

[err.log](https://github.com/user-attachments/files/17605406/err.log)
Commit hash: ed864bad570b94bf5e278566c7992ca22ddbdffb

I want to fix this hard coding

Reply:
Oh, you're on Darwin with Nix? That explains why libc++ was being used. Yeah, this hard-coding isn't great. We don't have good support for binary clang/llvm right now, so everything is assuming we're building from source.

For now, if it helps you get everything working, you can comment those out.

Reply:
Yes, I have been using `aarch64-darwin` platform with Nix

```txt
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 24.1.0, macOS 15.1`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.18.2`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/store/63lvc59rjz075kb1f11xww2x2xqpw1pn-source`
```

Now I am checking whether Jank-lang is compiled correctly with Nix in `x86_64-linux` platform, by UTM.

Reply:
For now, I will consider `x86_64-linux` and `aarch64-linux` as targets